docs/api/tray.md

```diff
@@ -194,7 +194,7 @@ Sets the `image` associated with this tray icon.
 
 #### `tray.setPressedImage(image)` _macOS_
 
-* `image` [NativeImage](native-image.md)
+* `image` ([NativeImage](native-image.md) | String)
```

Comment:
Instead of documenting everywhere that `NativeImage` can also be a `string` I think it makes more sense to remove this duplication and update `electron-typescript-definitions` to alias `NativeImage` to `NativeImage | String`

Reply:
Note this won't block this PR, just a thing we should probably do in the future